contrib/CMakeLists.txt

```diff
@@ -182,6 +182,28 @@ else()
     message(STATUS "Not using QPL")
 endif ()
 
+if (OS_LINUX AND ARCH_AMD64)
+    option (ENABLE_QATLIB "Enable Intel® QuickAssist Technology (Intel® QATlib)" ${ENABLE_LIBRARIES})
+elseif(ENABLE_QATLIB)
+    message (${RECONFIGURE_MESSAGE_LEVEL} "QAT library is only supported on x86_64 arch")
+endif()
+if (ENABLE_QATLIB)
+    option (ENABLE_USDM_DRV "A User Space DMA-able Memory (USDM) component which allocates/frees DMA-able memory" OFF)
```

Comment:
Can option `ENABLE_USDM_DRV` please be called `ENABLE_QAT_USDM_DRIVER`?

Reply:
Done